packages/cli/src/oclif/commands/test.js

```diff
@@ -0,0 +1,89 @@
+const { flags } = require('@oclif/command');
+
+const BaseCommand = require('../ZapierBaseCommand');
+const ValidateCommand = require('./validate');
+const constants = require('../../constants');
+const utils = require('../../utils');
+const { buildFlags } = require('../buildFlags');
+
+class TestCommand extends BaseCommand {
+  async perform() {
+    if (!this.flags['skip-validate']) {
```

Comment:
minor preference for flags being camel case rather than hyphenated so we can use `this.flags.skipValidate` instead. That's a breaking change though.

Reply:
Yeah, probably not worth it. Not only we have a lot of flags already using hyphens, but also most popular CLI tools (such as git and grep) follow this convention.